rasa/core/agent.py

```diff
@@ -183,12 +179,11 @@ def _load_and_set_updated_model(
                 "couldn't reach server. We'll retry later... "
                 "Error: {}.".format(e)
             )
-
             return None
 
 
 async def _run_model_pulling_worker(
-    model_server: EndpointConfig, wait_time_between_pulls: int, agent: "Agent"
```

Comment:
wait_time_between_pulls will be passed through the EndpointConfig right?

Reply:
Yes, changed it so that it is part of the EndpointConfig again.